### PR TITLE
Adds a parameter for the timestamp server url

### PR DESCRIPTION
**Pull request by dougbradbury**
I was having trouble with the default timeserver and wanted to be able
to use others. Particularly, the server that the vendor of my cert
recommends.

Also adds missing parameters to action.yml

### CODE REVIEW

**Comment by fbaptista** [comment]
We've created a similar WA for this. +1 to the PR

**Review by MurrayGroves — APPROVED** [review]
Also have the same problem. This seems like a good solution.

**Review by MurrayGroves — CHANGES_REQUESTED** [review]
The changes should be in `dist/index.js` not `index.ts`, since that's what `action.yml` runs. Changes in `index.ts` don't seem to do anything and I'm not sure of the file's purpose.

File: action.yml [hunk 1]
```diff
@@ -16,6 +16,16 @@ inputs:
...
     required: false
+  certificatesha1:
+    description: 'The sha1 or fingerprint of the certification'
+    required: false
+  certificatename:
+    description: 'The Name of the certificate'
+    required: false
+  timestampUrl:
+    description: 'URL of the timestamp server'
+    required: false
```
Comment (overheadhunter) [inline action.yml:27]:
```suggestion
    required: false
    default: 'http://timestamp.verisign.com/scripts/timstamp.dll'
```

**Review by overheadhunter — COMMENTED** [review]
The action.yml supports default values in machine-readable format as well.

**Comment by overheadhunter** [comment]
@Dana-Prajea This PR is open for a while now, do you plan to improve your awesome GH Action further? There are plenty forks already, but I'm a fan of merging changes back to the original project to avoid fragmentation.

**Comment by overheadhunter** [comment]
> The changes should be in `dist/index.js` not `index.ts`, since that's what `action.yml` runs. Changes in `index.ts` don't seem to do anything and I'm not sure of the file's purpose.

.ts will get transpiled to .js

**Review by krzysztofsitnik — APPROVED** [review]

**Comment by krzysztofsitnik** [comment]
Can you please merge it? Default timestamp is down.

**Pull request merged**